build.sbt

```diff
@@ -8,6 +8,13 @@ lazy val root = (project in file(".")).enablePlugins(PlayScala)
 scalaVersion := "2.12.2"
 
 libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "3.0.0" % Test
+//Database dependencies:
+libraryDependencies ++= Seq(
```

Comment:
If you want to organise them it could be nicer to do:

val dbDependencies = Seq(
  "com.typesafe.play" %% "play-slick" % "3.0.0",
  "com.typesafe.play" %% "play-slick-evolutions" % "3.0.0",
  "com.h2database" % "h2" % "1.4.195"
)

val testDependencies = Seq("org.scalatestplus.play" %% "scalatestplus-play" % "3.0.0" % Test)

libraryDependencies ++= dbDependencies ++ testDependencies